### PR TITLE
Update websites.json

### PR DESCRIPTION
**Pull request by MoritzKa**
### Describe the changes you have made in this PR

should contain SATurn and https://topup.vercel.app/

### Type of change (Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### How has this been tested?

I will test it with the generated built

### Checklist

- [ ] My code follows the style guidelines of this project and performed a self-review of my own code
- [ ] New and existing tests pass locally with my changes
- [ ] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by MoritzKa** [comment]
firefox_9CjAP30nlK
firefox_wIKvsWpXos

File: src/app/screens/Publishers/websites.json [hunk 1]
```diff
@@ -135,7 +135,13 @@
         "subtitle": "Earn sats for accomplishing tasks",
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/sats4likes.svg",
         "url": "https://www.sats4likes.com/"
-      }
+      },
+  	  {
```
Comment (bumi) [inline src/app/screens/Publishers/websites.json:139]: formatting needs to be fixed here. 
(2 spaces, no tabs)

**Review by bumi — APPROVED** [review]
actually saturn does not fit in there, it does not use webln and does not use the extension.

**Comment by MoritzKa** [comment]
I still need to test if Nakaphoto allows to sell music and video files.

**Comment by MoritzKa** [comment]
Changed to "images" only. 
I still cannot handle GitHub properly. Shall I create a new branch for each PR? Because this one does not seem to be up to date.

**Comment by escapedcat** [comment]
Relates to #1515

**Comment by MoritzKa** [comment]
firefox_ieSNBEWeah

**Review by escapedcat — APPROVED** [review]

**Pull request merged**